### PR TITLE
tests/robustness: Improve naming of Txn fields

### PR DESCRIPTION
**Pull request by serathius**
cc @ahrtr @jmhbnz @chaochn47

### CODE REVIEW

**Review by jmhbnz — APPROVED** [review]
Much easier to understand!

**Review by ahrtr — APPROVED** [review]
LGTM with a minor comment.

**Review by fuweid — APPROVED** [review]
LGTM

File: tests/robustness/model/deterministic.go [hunk 1]
```diff
@@ -264,8 +264,8 @@ type EtcdRequest struct {
 }
 
 type TxnRequest struct {
-	Conds []EtcdCondition
-	Ops   []EtcdOperation
+	Conditions []EtcdCondition
+	OnSuccess  []EtcdOperation
```
[the remark this answers is not in the document]
Comment (serathius) [reply tests/robustness/model/deterministic.go:268]: Done

**Pull request merged**